app/src/main/java/com/everest/moviedb/model/MovieRepository.kt

```diff
@@ -0,0 +1,9 @@
+package com.everest.moviedb.model
+
+class MovieRepository constructor(private val movieService: MovieService) {
+    fun getAllMovies() = movieService.getAllMovies()
+
+    fun getCurrentPlayingMovies() = movieService.getCurrentPlayingMovies()
```

Comment:
we should enqueue the call here only return the response or error from here
check sealed classes to handle/return it easily